### PR TITLE
Use pouchdb memory adapter on iOS

### PR DESCRIPTION
**Pull request by justinbarry**
Using the SingleThreadedConnector was breaking some wallets on mobile. Roping in @andrewdamelio to help me ensure this isn't the case here.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🌶

 This link will always point to this PR's latest changes: https://peek.run/kq3h274

File: packages/augur-ui/src/services/augursdk.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export class SDK {
       account
     );
 
-    const connector = (new WebWorkerConnector());
+    const connector = (iOSSafari ? new SEOConnector(): new WebWorkerConnector());
```
Comment (bthaile) [inline packages/augur-ui/src/services/augursdk.ts:53]: do all the eventing/emitting work on SEOConnector?

**Review by andrewdamelio — APPROVED** [review]
LGTM - works on mobile

File: packages/augur-ui/src/services/augursdk.ts (hunk 1 above)
Comment (justinbarry) [reply packages/augur-ui/src/services/augursdk.ts:53]: Yes. See attached.
image

**Pull request merged**